src/main/java/com/google/jenkins/plugins/k8sengine/KubernetesEnginePublisher.java

```diff
@@ -141,6 +155,10 @@ public boolean isVerifyServices() {
     return this.verifyServices;
   }
 
+  public String isEntryMethod(String entryMethodName) {
+    return this.entryMethod.equalsIgnoreCase(entryMethodName) ? "true" : "";
```

Comment:
What happens if this method is called before setEntryMethod(), would it be worth adding a null check?

Reply:
When I tested this, there weren't any error messages in the terminal where I ran it. Is it possible that setEntryMethod() is called with the empty string once the plugin is loaded in the post build actions?

This might also be a good reason to avoid the uncertainty and add a default value. Would there be any issues with initializing it in the constructor?

Reply:
Sure that works.